Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -253,6 +253,11 @@ extension HTTPClient {
             return self.scheme == "https" || self.scheme == "https+unix"
         }
 
+        /// Specified port.
+        public var specifiedPort: Int? {
```

Comment:
Do we need to expose this in public API?

Reply:
ah we don't need that I guess